client/components/wizard/README.md

```diff
@@ -0,0 +1,62 @@
+Wizard
+======
+`Wizard` is a component that leads the user through a series of pre-defined steps.
+It keeps track of progress and enables navigating back or forward through the steps.
+
+## Usage
+
+```js
+const components = {
+	'first': <First />,
+	'second': <Second />,
+};
+const steps = [ 'first', 'second' ];
+
+<Wizard
+	basePath="/section/wizard"
+	components={ components }
+	steps={ steps }
```

Comment:
Would be neat to drop steps and just rely on `components`' items order. Is that order stable across browsers these days?